search-service/src/test/kotlin/com/egm/stellio/search/service/AttributeInstanceServiceTests.kt

```diff
@@ -268,19 +283,39 @@ class AttributeInstanceServiceTests : TimescaleBasedTests() {
             )
         )
 
-        val attributeInstance =
-            attributeInstanceService.addAttributeInstances(temporalEntityAttribute.id, "outgoing", attributeValues)
+        attributeInstanceService.addAttributeInstances(
+            temporalEntityAttribute.id,
+            "outgoing",
+            attributeValues,
+            parsedObservationPayload
```

Comment:
I have a doubt. What is persisted:

```
{
          "outgoing": {
            "type": "Property",
            "value": 550.0
          }
}
```

or 

```
{
            "type": "Property",
            "value": 550.0
}
```

Reply:
persisted in the attribute instance payload ?
```
 {
            "type": "Property",
            "value": 550.0
}
```
the assertion is [here](https://github.com/stellio-hub/stellio-context-broker/pull/318/files/930fe27de8d2c85ad40b11fb098925d498ea1720#diff-f78a686e2a32b3527690afee212dbf7d7da69c2f9600c6380aecece0d6dce6d1R298)